docs/source/en/index.mdx

```diff
@@ -209,6 +209,7 @@ Flax), PyTorch, and/or TensorFlow.
 |            GLPN             |       ❌       |       ❌       |       ✅        |         ❌         |      ❌      |
 |           GPT Neo           |       ❌       |       ❌       |       ✅        |         ❌         |      ✅      |
 |            GPT-J            |       ❌       |       ❌       |       ✅        |         ✅         |      ✅      |
+|          greaselm           |       ❌       |       ❌       |       ✅        |         ❌         |      ❌      |
```

Comment:
```suggestion
|          GreaseLM           |       ❌       |       ❌       |       ✅        |         ❌         |      ❌      |
```
This was probably a mistake in the CookieCutter, should be just the name of the model here.